### ISSUE TITLE
models bert pretrain model variable name is not the name with original bert?

### ISSUE THREAD

**Issue by RyanHuangNLP** [tensorflow/models#7479]
original [bert](https://github.com/google-research/bert) pretrain model variable name is not the same with model [bert](https://github.com/tensorflow/models/tree/master/official/bert), could you open source the convert script?

**Comment by saberkun** [comment]
Yes, they are different, basically because of the organization of keras layers.
Of course, we can open source the conversion which is quite simple.
We released a few checkpoints. Are you using your own TF1 checkpoints?

**Comment by RyanHuangNLP** [comment]
@saberkun 
no, I use the google release, but I want to use chinese bert that not provide in the models one

`var_model_bert = tf.train.list_variables('cased_L-12_H-768_A-12/bert_model.ckpt')` and `var_original_bert = tf.train.list_variables('cased_L-12_H-768_A-12/bert_model.ckpt')`, after I load the checkpoint file with the model one and original one, I guess that the model one slice `var_model_bert[1:-1]` and original one slice `var_original_bert[:199]` may be have same parameter but different variable name.

if my guess is right, is it just change the variable name will fix this issue.

**Comment by saberkun** [comment]
you are right. Just names are different, variable shapes can be exactly matched. there is some tricks to directly load tf1 checkpoint in tf2. I will open source the converter first.
what do you by removing cls parameter?

**Comment by RyanHuangNLP** [comment]
@saberkun 
I find this variable names:
```
[('cls/predictions/output_bias', [28996]),
 ('cls/predictions/transform/LayerNorm/beta', [768]),
 ('cls/predictions/transform/LayerNorm/gamma', [768]),
 ('cls/predictions/transform/dense/bias', [768]),
 ('cls/predictions/transform/dense/kernel', [768, 768]),
 ('cls/seq_relationship/output_bias', [2]),
 ('cls/seq_relationship/output_weights', [2, 768])]
```
is no need in the models one, is it not neccesary in the downstream task?

**Comment by RyanHuangNLP** [comment]
@saberkun when would you open source the convert, I would like to know the trick about convert the checkpoint files
